frontend/packages/kubevirt-plugin/src/components/vm-nics/create-nic-row.tsx

```diff
@@ -157,12 +158,26 @@ export const CreateNicRow: React.FC<CreateNicRowProps> = ({
         />
       </TableData>
       <TableData className={dimensify()}>
-        <Text
-          id="nic-mac-address"
-          onChange={setMacAddress}
-          value={macAddress}
-          disabled={creating || networkType === NetworkType.POD}
-        />
+        <FormGroup
+          className="kubevirt-vm-create-device-row__cell--no_bottom"
+          validationState={
+            macAddressError && macAddressError.type === ValidationErrorType.Error
+              ? macAddressError.type
```

Comment:
this should be a message, right? Can you please create a function for this and use it also in the name?

Reply:
This shouldn't be a message, no. It should be an error type. The message property is used further down in a similar block. I can create functions for both, though.

Reply:
ah, you are right, sorry. I forgot this is an pf3 FormGroup. We should port these to pf4 soon.

>  I can create functions for both, though.

thanks

Reply:
Done!